examples/conference.ipynb

```diff
@@ -0,0 +1,287 @@
+            "text/plain": [
+              "<Figure size 432x288 with 1 Axes>"
+            ]
+          },
+          "metadata": {
+            "needs_background": "light"
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Run DP pipeline\n",
+        "Below we compute the same statistics using differential privacy and PipelineDP."
+      ],
+      "metadata": {
+        "id": "IIjQrB3eFmvp"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "id": "eN9fu0NkSA6u",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "15bf4d5c-6a2e-48a0-8830-81c1a5ecbab4"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "[('Germany', MetricsTuple(privacy_id_count=49.10272994384104)), ('Switzerland', MetricsTuple(privacy_id_count=75.26271629976691)), ('France', MetricsTuple(privacy_id_count=32.206397102141636)), ('Italy', MetricsTuple(privacy_id_count=37.134226348807715)), ('UK', MetricsTuple(privacy_id_count=97.95130274764233))]\n"
+          ]
+        }
+      ],
+      "source": [
+        "#@title DP statistics\n",
+        "\n",
+        "# Choose the backend: local, Beam or Spark\n",
+        "backend = pipeline_dp.LocalBackend()\n",
+        "\n",
+        "# Define the total privacy loss that can be introduced by this pipeline\n",
+        "budget_accountant = pipeline_dp.NaiveBudgetAccountant(total_epsilon=1, total_delta=1e-6)\n",
+        "\n",
+        "# Create DPEngine\n",
+        "dp_engine = pipeline_dp.DPEngine(budget_accountant, backend)\n",
+        "\n",
+        "# Configure functions to extract partition key, privacy ID and aggregated value\n",
+        "# from the input data\n",
+        "data_extractors = pipeline_dp.DataExtractors(\n",
+        "   partition_extractor=lambda row: row[1],\n",
+        "   privacy_id_extractor=lambda row: row[0],\n",
+        "   value_extractor=lambda row: 1)\n",
+        "\n",
+        "# Configure the aggregation parameters\n",
+        "params = pipeline_dp.AggregateParams(\n",
+        "   noise_kind=pipeline_dp.NoiseKind.LAPLACE,\n",
+        "   metrics=[pipeline_dp.Metrics.PRIVACY_ID_COUNT],\n",
+        "   max_partitions_contributed=1,\n",
+        "   max_contributions_per_partition=1,\n",
+        "   min_value=0,\n",
+        "   max_value=1)\n",
+        "\n",
+        "# Create a computational graph for the aggregation.\n",
+        "# All computations are lazy. dp_result is iterable, but iterating it would\n",
+        "# fail until budget is computed (below).\n",
+        "# It’s possible to call DPEngine.aggregate multiple times with different\n",
+        "# metrics to compute.\n",
+        "dp_result = dp_engine.aggregate(input, params, data_extractors)\n",
+        "\n",
+        "# Compute budget per each DP operation. \n",
```

Comment:
nit: delete period at the end for consistency (or alternatively, put periods at the end of other comments.